pkg/syncthing/syncthing.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syncthing
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/syncthing/syncthing/lib/config"
+	"github.com/syncthing/syncthing/lib/protocol"
+)
+
+type Syncthing struct {
+	Name          string
+	Cmd           *exec.Cmd
+	Config        *config.Configuration
+	PrevConfig    config.Configuration // Unapplied config
+	HomeDirectory string
+	Port          string
+	DeviceID      protocol.DeviceID
+	Client        *Client
+	ApiKey        string
+	latestEventId int64
+	DeviceAddress string
+}
+
+// Initializes the remote syncthing instance
+func InitializeRemoteSyncthing() (*Syncthing, error) {
+	s := &Syncthing{
+		Name:          "Remote Syncthing",
+		Port:          DefaultRemotePort,
+		HomeDirectory: "/config",
```

Comment:
Any special reason to use this uncommon path?

Reply:
This is the default for the syncthing image.